runtime/tests/runtime_integration_tests/common.rs

```diff
@@ -189,6 +191,10 @@ pub const fn svn_to_bitmap(svn: u32) -> [u32; 4] {
     ]
 }
 
+pub fn rom_for_fw_integration_tests() -> io::Result<Cow<'static, [u8]>> {
+    caliptra_builder::rom_for_fw_integration_tests_fpga(cfg!(feature = "fpga_subsystem"))
```

Comment:
Shouldn't this also be set to true for `fpga_realtime`:


```suggestion
    caliptra_builder::rom_for_fw_integration_tests_fpga(cfg!(any(feature = "fpga_realtime", feature = "fpga_subsystem")))
```